src/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Greeting from './features/greeting/Greeting';
+
+function App() {
+  return (
+    <div>
+      <Greeting />
+    </div>
+  );
+}
```

Comment:
- You have done well so far, but according to the project requirements, your app component should have a route that leads to the Greeting component. Please wrap your Greeting component in a route as shown below 👍 
```javascript
   function App() {
     return (
       <div className="App">
         <Routes>
           <Route path="/" element={<Greeting />} />
         </Routes>
       </div>
  );
}
```
<img width="562" alt="reqewraewrawe" src="https://github.com/fmanimashaun/hello-react-front-end/assets/73607512/ac55b135-7b3b-4bfb-81a8-04227affe28d">

Reply:
resolved